parser/src/java/org/apache/hadoop/hive/ql/parse/AlterClauseParser.g

```diff
@@ -631,3 +631,48 @@ alterDataConnectorSuffixSetUrl
     -> ^(TOK_ALTERDATACONNECTOR_URL $dcName $newUri)
     ;
 
+alterScheduledQueryStatement
```

Comment:
Could you please descripe the detail about HIVE-28384 `Antler code too long issue`.

Reply:
Thanks @kokila-19 for adding details for the error occupied here in the ticket.